cmd/kar-controllers/app/options/options.go

```diff
@@ -48,6 +50,8 @@ func (s *ServerOption) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.Kubeconfig, "kubeconfig", s.Kubeconfig, "Path to kubeconfig file with authorization and master location information.")
 	fs.BoolVar(&s.Dispatcher,"dispatcher",s.Dispatcher,"set dispather mode(true) or agent mode(false)")
 	fs.StringVar(&s.AgentConfigs, "agentconfigs", s.AgentConfigs, "Paths to agent config file:deploymentName separted by commas(,)")
+	fs.BoolVar(&s.DynamicPriority,"dynamicpriority", s.DynamicPriority,"--dynamicpriority=false/true set controller use dynamic priority")
```

Comment:
Minor description update suggestion: `--dynamicpriority=false/true set controller to use dynamic priority`.  Added `to` in the description.  
Since now we will have multiple policies, may add some additional text with regards to what policy is used when `dynamicpriority=false`?  Please let me know your input.  Thanks!